src/Tribe/Metabox.php

```diff
@@ -195,7 +195,7 @@ public function ajax_ticket_add() {
 		$data = wp_parse_args( tribe_get_request_var( array( 'data' ), array() ), array() );
 
 		if ( ! $this->has_permission( $post_id, $_POST, 'add_ticket_nonce' ) ) {
-			wp_send_json_error( esc_html__( 'Failed to Add the Ticket, Refresh the Page to try again.', 'event-tickets' ) );
+			wp_send_json_error( sprintf( esc_html__( 'Failed to add the %s. Refresh the page to try again.', 'event-tickets' ), tribe_get_ticket_label_singular( 'ajax_ticket_add_error' ) ) );
```

Comment:
This only escapes the __ string, not the strings you are inserting via printf(). Can you follow the other examples where I've made suggestions that does it like `esc_html( sprintf( .... ) )` here elsewhere in your PR?